horizomer/utils/tests/test_tree.py

```diff
@@ -361,6 +361,22 @@ def test_is_ordered(self):
         tree5 = TreeNode.read(['((h,(e,g)),((a,b),(c,d,i)j));'])
         self.assertTrue(is_ordered(tree5, False))
 
+    def test_cladistic(self):
+        tree1 = TreeNode.read(['((i,j)a,b)c;'])
```

Comment:
why don't you define trees in setUp, instead? I see you're using the same tree in `test_ordered_nodes`.

Reply:
That's because this whole script has dozens of functions and even more test trees, most of which are distinct. It is just a rare coincidence that these two functions used the same tree.